charts/coolify/templates/coolify-app-configmap.yaml

```
@@ -0,0 +1,88 @@
+  LOG_CHANNEL: "single"
+  LOG_DEPRECATIONS_CHANNEL: "null"
+  LOG_LEVEL: "info"
+  
+  # Laravel Broadcasting Configuration
+  BROADCAST_DRIVER: "pusher"
+  
+  # Laravel Session Configuration
+  SESSION_SECURE_COOKIE: "false"
+  SESSION_HTTP_ONLY: "true"
+  SESSION_SAME_SITE: "lax"
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)